circle.yml

```diff
@@ -0,0 +1,18 @@
+machine:
+  environment:
+    IMPORT_PATH: "github.com/gruntwork-io/${CIRCLE_PROJECT_REPONAME}"
+    REPO: "$HOME/.go_workspace/src/${IMPORT_PATH}"
+    GOOS: linux
+
+dependencies:
+  override:
+    # Get our repo into the CircleCI GOPATH.
+    - |
+        mkdir -p ${REPO}
+        rmdir ${REPO}
+        ln -s "/home/ubuntu/${CIRCLE_PROJECT_REPONAME}" "${REPO}"
+
+test:
+  override:
+    - |
+        ${REPO}/_ci/run-tests.sh
```

Comment:
I don't want to switch over to glide for one dependency, so I tried to just run `go test -i`. https://github.com/brikis98/ec2-snapper/commit/51c8f761db78037038d590f0d5a976b85ad6754c